src/datreant/core/trees.py

```diff
@@ -469,25 +469,36 @@ def make(self):
 
         return self
 
-    def sync(self, dest, mode='upload', compress=True, backup=False, dry=False,
-             include=None, exclude=None, rsync_path='/usr/bin/rsync'):
+    def sync(self, other, mode='upload', compress=True, checksum=True,
+             backup=False, dry=False, include=None, exclude=None,
+             rsync_path='/usr/bin/rsync'):
         """Syncronize directories using rsync.
 
         Parameters
         ----------
-        dest: destination
+        other: str or Tree
+            Other end of the sync, can be either a path or another Tree.
+        mode: str
+            Either ``"upload"`` if uploading to  *other*, or ``"download"`` if
+            downloading from *other*
```

Comment:
Use single backticks for `other` here, since it's an arg name; this is one of the numpy doc conventions